x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SnapshotLifecycleService.java

```diff
@@ -234,6 +236,20 @@ public static void validateRepositoryExists(final String repository, final Clust
             .orElseThrow(() -> new IllegalArgumentException("no such repository [" + repository + "]"));
     }
 
+    /**
+     * Validates that the interval between snapshots is not smaller than the minimum interval
+     * (see {@link LifecycleSettings#SLM_MINIMUM_INTERVAL_SETTING})
+     * @throws IllegalArgumentException if the interval is less than the minimum
+     */
+    public static void validateMinimumInterval(final SnapshotLifecyclePolicy lifecycle, final ClusterState state) {
```

Comment:
Since it's static (yay), can you add a unit test for this method to `SnapshotLifecycleServiceTests`?

Reply:
✅, 4526dc70c1f13659fb0a0c5ef596c8a660280098